public/app/ui/components/search/results/search-results.html

```diff
@@ -36,8 +36,8 @@
                     <tbody>
                     <!-- ko foreach: { data: results, as: 'result' } -->
                     <tr class="result">
-                        <td data-bind="text: $index() + $parent.pager.start() + 1"></td>
-                        <!-- ko foreach: { data: $parent.fields, as: 'field' } -->
+                        <td data-bind="text: $index() + $parents[1].pager.start() + 1"></td>
```

Comment:
I'll make the change - be good to know how it ticks. Thanks for the explanation and enjoy your holiday ;)